swift_test.go

```diff
@@ -68,8 +70,14 @@ func TestTransport(t *testing.T) {
 		ApiKey = "swifttest"
 		AuthUrl = srv.AuthURL
 	}
-	tr := &someTransport{Transport: http.Transport{MaxIdleConnsPerHost: 2048}}
+	tr := &someTransport{
```

Comment:
This looks like testing code?  We certainly don't want `InsecureSkipVerify` on by default